index.js

```diff
@@ -25,14 +25,14 @@ function encodePair (msg) {
 
   if(isString(value)) {
     flags = STRING
-    value = new Buffer(value)
+    value = Buffer.from(value)
```

Comment:
Sometimes I'm really specific and say `Buffer.from(value, 'utf8')`

This is the default but I feel more secure clearly communicating to readers and locking it down?